guides/actions_and_events_overview.md

```diff
@@ -0,0 +1,16 @@
+# Actions and Events
```

Comment:
Remove this file in lieu of the Actions and Events doc I've got

Reply:
Removed as suggested.